packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -235,7 +247,7 @@ export const useArbTokenBridge = (
   }) {
     try {
       const ethBridger = await EthBridger.fromProvider(l2.provider)
-
+      const walletAddress = await l2Signer.getAddress()
```

Comment:
Why aren't we reusing `address` derived from `useAccount` here, and in the other functions also?

Reply:
`withdrawEth` is not a hook, so we can't call `useAccount` here.

Reply:
What I meant was using `address` derived from `useAccount` in the `useArbTokenBridge` hook in line 99 in these functions as well. Since `address` has already been defined outside the function, why not use it inside as well instead of freshly deriving it in this function which also adds an await statement inside, unless there is a specific reason to do so which I might be unaware of?

We are also using `addTransaction` function defined outside these functions inside these.